conf/routes

```diff
@@ -4,112 +4,205 @@ GET        /index.html                 @org.sagebionetworks.bridge.play.controll
+GET    /api/v1/profile                   @org.sagebionetworks.bridge.play.controllers.UserProfileController.getUserProfileV1
+POST   /api/v1/profile                   @org.sagebionetworks.bridge.play.controllers.UserProfileController.updateUserProfileV1
 
 # API - User Settings
-POST   /api/v1/profile/external-id       @org.sagebionetworks.bridge.play.controllers.UserProfileController.createExternalIdentifier
-POST   /api/v1/profile/unsubscribe       @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmail
-GET    /api/v1/profile/unsubscribe       @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmail
+POST   /api/v1/profile/external-id       @org.sagebionetworks.bridge.play.controllers.UserProfileController.createExternalIdentifierV1
+POST   /api/v1/profile/unsubscribe       @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmailV1
+GET    /api/v1/profile/unsubscribe       @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmailV1
 
 # API - Surveys & Survey Response v2
-GET    /api/v2/surveyresponses/:guid                                @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.getSurveyResponse(guid: String)
-POST   /api/v2/surveyresponses/:guid                                @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.appendSurveyAnswers(guid: String)
-GET    /api/v2/surveys/:surveyGuid/revisions/published              @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyMostRecentlyPublishedVersionForUser(surveyGuid: String)
-GET    /api/v2/surveys/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyForUser(surveyGuid: String, createdOn: String)
-POST   /api/v2/surveys/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponse(surveyGuid: String, createdOn: String)
-POST   /api/v2/surveys/:surveyGuid/revisions/:createdOn/:identifier @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponseWithIdentifier(surveyGuid: String, createdOn: String, identifier: String)
+GET    /api/v2/surveyresponses/:guid                                @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.getSurveyResponseV1(guid: String)
+POST   /api/v2/surveyresponses/:guid                                @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.appendSurveyAnswersV1(guid: String)
+GET    /api/v2/surveys/:surveyGuid/revisions/published              @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyMostRecentlyPublishedVersionV1(surveyGuid: String)
+GET    /api/v2/surveys/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyV1(surveyGuid: String, createdOn: String)
```

Comment:
getSurveyMostRecentlyPublishedVersionV1 and getSurveyV1 have the same name as their researcher counterparts. Will this work?